### DISCUSSION TITLE
Strange Z numbers and other gremlins

### DISCUSSION THREAD

**Discussion by rorywquin** [Jyers/Marlin#104] [Q&A] [answered]
Installed your FW -  BLT 5x5 Fast for the 4.2.2 board

1) Was following this video (https://www.youtube.com/watch?v=y_1Kg45APko) to get my Z axis closer to the bed (seems 10mm is standard in your setup) change home zero). At the point of lowering the nozzle to a piece of paper, the z value ( at the very bottom of the screen) slowly comes down toward zero and then below zero becomes a massive number. Around 700 which gets smaller as I lower the nozzle. Managed to work around it by counting the (.1mm) increments with using Pronterface. Expected to read a negative value of ~2mm.

2) EDIT: Solved the other Gremlin not related to this product.

Cheers

**Comment by Jyers** [comment, marked as answer]
I'm assuming your referring to changing the Z offset. May I ask why you are doing this via Pronterface rather than using the on screen Z offset menu? Either way I see the issue, the z pos value is not coded to be able to go below zero, so I will most likely modify this to simply read zero when it is negative rather than the strange number.

**Reply by rorywquin** [reply]
1) Installed your FW.
2) Homed Z and it was about 15mm from teh bed.
3) This meant bed was too far do manual levelling (limitation of the adjusters)
4) So wanted to get the nozzle closer to the bed.
5) Had successfully used pronterface before to get a new Zero (home) point for Nozzle
6) Using the builtin z-off set tool (used it regularly in other FW) would have resulted in an offset of ~ -15mm.......wouldn't it?

Not sure where I was wrong?

**Reply by Jyers** [reply]
After homing the z axis is raised up. So the 15mm position you noted is not actually Z=0. Try using the Z offset menu as it will auto zero the Z axis after home and move the axis as you adjust so you can easily set up your offset.
